daphne/src/roles/mod.rs

```diff
@@ -1103,49 +1103,6 @@ mod test {
 
     async_test_versions! { handle_agg_job_req_transition_continue }
 
```

Comment:
Why delete this test?

Reply:
Bump: As discussed on Tuesday, we ought to rewrite this test rather than delete it. All that's changed is that we don't detect replays until just before completing the aggregation job. In draft02, this is on the continue request; in the latest draft, this is on the initial request.